core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java

```diff
@@ -90,4 +124,137 @@ void entityWithoutType() {
+            context.translate(
+                playerInfoUpdateTranslator,
+                new ClientboundPlayerInfoUpdatePacket(
+                    EnumSet.of(PlayerListEntryAction.ADD_PLAYER, PlayerListEntryAction.UPDATE_LISTED),
+                    new PlayerListEntry[] {
+                        new PlayerListEntry(npcUuid, new GameProfile(npcUuid, "1297"), false, 0, GameMode.SURVIVAL, null, null, 0, null, null)
+                    }));
+
+            //todo we don't have to remove an entry that was never in the playerlist in the first place
+            assertNextPacket(context, () -> {
+                var packet = new PlayerListPacket();
+                packet.getEntries().add(new PlayerListPacket.Entry(npcUuid));
+                packet.setAction(PlayerListPacket.Action.REMOVE);
+                return packet;
+            });
+            assertNoNextPacket(context);
+
+            context.translate(
+                setPlayerTeamTranslator,
+                new ClientboundSetPlayerTeamPacket(
+                    "npc_team_1297",
+                    Component.empty(),
+                    Component.empty(),
+                    Component.empty(),
+                    false,
+                    false,
+                    NameTagVisibility.NEVER,
+                    CollisionRule.NEVER,
+                    TeamColor.WHITE,
+                    new String[0]
+                )
+            );
+            context.translate(
+                setPlayerTeamTranslator,
+                new ClientboundSetPlayerTeamPacket("npc_team_1297", TeamAction.ADD_PLAYER, new String[]{ "1297" }));
+
+            context.translate(addEntityTranslator, new ClientboundAddEntityPacket(1297, npcUuid, EntityType.PLAYER, 1, 2, 3, 4, 5, 6));
+            // then it updates the displayed skin parts, which isn't relevant for us
+
+            assertNextPacketMatch(context, AddPlayerPacket.class, packet -> {
+                assertEquals(3, packet.getRuntimeEntityId());
+                assertEquals(npcUuid, packet.getUuid());
+                assertEquals("1297", packet.getUsername());
+                assertEquals((byte) 1, packet.getMetadata().get(EntityDataTypes.NAMETAG_ALWAYS_SHOW));
+                assertEquals("", packet.getMetadata().get(EntityDataTypes.NAME));
+            });
+            assertNoNextPacket(context);
+
+
+            // second command, create hologram
+
+
+            var hologramUuid = UUID.fromString("b1586291-5f68-44dc-847d-6c123c5b8cbf");
+            context.translate(
+                addEntityTranslator,
+                new ClientboundAddEntityPacket(1298, hologramUuid, EntityType.ARMOR_STAND, 6, 5, 4, 3, 2, 1));
+
+            assertNextPacketMatch(context, AddEntityPacket.class, packet -> {
+                assertEquals(4, packet.getRuntimeEntityId());
+                assertEquals("minecraft:armor_stand", packet.getIdentifier());
+            });
+
+            // invisible, custom name, custom name visible
```

Comment:
Sorry, what does this mean? is that just a visibility change from invisible -> visible custom name?

Reply:
It is a comment for the Entity metadata I send. Index 0 with 0x20 is the flag for invisibility. Index 2 stands for the custom name. Index 3 stands for custom name visible.